### DISCUSSION TITLE
Availability of v9.23 on npm?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#17659] [Archive]
First let me thank you for the great work done for [v9.23: 79 issues/improvements resolved](https://github.com/directus/directus/milestone/136?closed=1) Wooh!

It looks like it did not go to npm: [at line 45](https://github.com/directus/directus/actions/runs/4295889524/jobs/7486979867#step:5:46) there is an `npm ERR! code ENEEDAUTH` due to some recent changes in npm explained [here](https://stackoverflow.com/questions/75357158/github-actions-npm-publish-fails-with-err-code-eneedauth).

**Comment by br41nslug** [comment]
A fix has been pushed, we'll have to re-release. https://github.com/directus/directus/pull/17656
